functions/Backup-DbaDatabase.ps1

```diff
@@ -71,27 +75,47 @@ Backs up AdventureWorks2014 to sql2016's C:\temp folder
 #>
 	[CmdletBinding(DefaultParameterSetName = "Default")]
 	param (
-		[parameter(ParameterSetName = "NoPipe", Mandatory = $true)]
+		[parameter(ParameterSetName = "NoPipe", Mandatory = $true)][parameter(ParameterSetName = "Pipe")]
 		[object[]]$SqlInstance,
+		[parameter(ParameterSetName = "NoPipe")][parameter(ParameterSetName = "Pipe")]
 		[System.Management.Automation.PSCredential]$SqlCredential,
-		[string]$BackupDirectory,
+		[parameter(ParameterSetName = "NoPipe")][parameter(ParameterSetName = "Pipe")]
+		[string[]]$BackupDirectory,
+		[parameter(ParameterSetName = "NoPipe")][parameter(ParameterSetName = "Pipe")]
 		[string]$BackupFileName,
+		[parameter(ParameterSetName = "NoPipe")][parameter(ParameterSetName = "Pipe")]
 		[switch]$NoCopyOnly,
+		[parameter(ParameterSetName = "NoPipe")][parameter(ParameterSetName = "Pipe")]
 		[ValidateSet('Full', 'Log', 'Differential', 'Diff', 'Database')]
 		[string]$Type = "Database",
-		[parameter(ParameterSetName = "Pipe", Mandatory = $true, ValueFromPipeline = $true)]
-		[object[]]$DatabaseCollection
+		[parameter(ParameterSetName = "Pipe",Mandatory = $true, ValueFromPipeline = $true)]
+		[object[]]$Arrdatabases,
```

Comment:
Hey @Stuart-Moore - naming is hard and in PS v4, we could actually hide this from the end user, but unfortunately in v3 we can't. Once we get TabExtension++ setup, this will actually be $Database but for now, please keep it DatabaseCollection as DatabaseCollection  is more PowerShelly than Arrdatabases.

In addition, we're getting Pester failures for missing help. Please add help for CreateFolder and SqlInstance.